NeosModLoader/NeosModLoader.csproj

```diff
@@ -28,6 +29,14 @@
 		<NeosPath Condition="Exists('D:/Files/Games/Neos/app/')">D:/Files/Games/Neos/app/</NeosPath>
 	</PropertyGroup>
 
+	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+	  <DebugType>none</DebugType>
+	</PropertyGroup>
+
+	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+	  <DebugType>embedded</DebugType>
+	</PropertyGroup>
```

Comment:
Rather would just do `Condition="'$(Configuration)'=='Debug'"` to avoid any issues with possibly configuring other Platforms

Reply:
that's just what VS generated, but will see 🤔